src/Mollie/WC/Plugin.php

```diff
@@ -423,7 +426,51 @@ public static function saveProductVoucherOptionFields($post_id)
         );
     }
 
+    /**
+     * Add dedicated voucher category field for variations.
+     * Default is the same as the general voucher category
+     * @param $loop
+     * @param $variation_data
+     * @param $variation
+     */
+    public static function voucherFieldInVariations($loop, $variation_data, $variation ) {
```

Comment:
It looks like `$loop` and `$variation_data` are not used in the body of he method.